.coveragerc

```diff
@@ -7,4 +7,3 @@ omit =
      */diffsims/utils/scattering_params.py
      */diffsims/version.py
      */diffsims/utils/__init__.py
-     */diffsims/tests/*
```

Comment:
Just to clarify - you're removing the tests from the omit list, i.e. they will now be included in coverage? If so, that sounds good to me, just checking...

Reply:
Yep, the coveralls on this branch includes the tests